src/sections/General/Navigation/navigation.style.js

```diff
@@ -374,6 +374,17 @@ const NavigationWrap = styled.header`
       max-height: 400px;
       overflow-y: scroll;
```

Comment:
You need to add `scrollbar-width: thin` to cover other browsers as well.

Reply:
@Nikhil-Ladha does it look good now?